Makefile

```diff
@@ -56,30 +56,13 @@ utest:
 		github.com/Oneledger/protocol/rpc \
 		github.com/Oneledger/protocol/identity \
 		github.com/Oneledger/protocol/app \
+		github.com/Oneledger/protocol/action/staking \
```

Comment:
Writing unit tests in here takes lots of time on setting up the environment. We are now testing Txs by using integration tests located in protocol/scripts/ folder

Reply:
How this is related to integration tests?